LINES

```diff
@@ -0,0 +1,5 @@
+C F G Bb C Db Eb F E . . . . . .
```

Comment:
A little (possibly parseable) syntax for describing lines (in the key of C), the idea I guess is that `-` means a rest and `.` means sustaining the previous note (and everything is describing sixteenth notes)

This first one is the line I hardcoded for now (without as long of a sustain of the last note)